src/Nelmio/Alice/Persister/Doctrine.php

```diff
@@ -15,17 +15,31 @@
 use Nelmio\Alice\PersisterInterface;
 
 /**
- * The Doctrine persists the fixtures into an ObjectManager
+ * Adaptor for the ObjectManager to provide a Doctrine bridge.
+ *
+ * @author Jordi Boggiano <j.boggiano@seld.be>
+ * @author Robert Schönthal
  */
 class Doctrine implements PersisterInterface
 {
-    protected $om;
+    /**
+     * @var ObjectManager
+     */
+    protected $manager;
+
+    /**
+     * @var bool
+     */
     protected $flush;
 
-    public function __construct(ObjectManager $om, $doFlush = true)
+    /**
+     * @param ObjectManager $manager
+     * @param bool          $flush
+     */
+    public function __construct(ObjectManager $manager, $flush = true)
```

Comment:
As a Doctrine user I find that more confusing than anything. But then again changing this causes some BC break. Not in the library itself neither for people using `Fixtures`. So theatrically it may break something for someone.

We can also keep the current behaviour for now and add a deprecation notice here too (I'm getting the feeling I'm only talking about deprecation tonight).